README.md

```diff
@@ -82,11 +82,14 @@ The `aws-crt` JAR in Maven Central is a large "uber" jar that contains compiled
 
 ### Available platform classifiers
 
-- linux-armv6
-- linux-armv7
+- linux-armv6 (no auto-detect)
+- linux-armv7 (no auto-detect)
 - linux-aarch_64
 - linux-x86_32
 - linux-x86_64
+- linux-x86_64-musl (no auto-detect)
+- linux-armv7-musl (no auto-detect)
+- linux-aarch_64-musl (no auto-detect)
```

Comment:
I see that the os-maven-plugin doesn't detect musl, but is there a classifier that other Java projects have settled on? I googled and stumbled across `linux_musl` being used as the platform [here](https://jreleaser.org/guide/latest/tools/jdks-maven.html), so it would be like `linux_musl-x86_64` instead of ~linux-x86_64-musl~

can you do some more research on this?

Reply:
if we do end up going with linux_musl, I wonder if it makes more sense to consider "linux_musl" its own platform everywhere else in our code, so we can continue with `platform/arch` paradigm instead of changing to `platfom/arch/cruntime` paradigm

e.g. organize the files like:
```
linux/x86_64/libaws-crt-jni.dylib
linux_musl/x86_64/libaws-crt-jni.dylib
osx/x86_64/libaws-crt-jni.dylib
```

instead of 
```
linux/x86_64/glibc/libaws-crt-jni.dylib
linux_musl/x86_64/musl/libaws-crt-jni.dylib
osx/x86_64/cruntime/libaws-crt-jni.dylib
```

Reply:
I couldn't find much information on which classifiers is common. I found [tokyocabinet-linux_musl-x64/0.0.14](https://mvnrepository.com/artifact/com.fizzed/tokyocabinet-linux_musl-x64/0.0.14) which uses the `linux_musl` classifier, and [rocksdbjni](https://mvnrepository.com/artifact/org.rocksdb/rocksdbjni) which uses `linux-x86_64-musl`. Looking at the documentation for the [os-maven-plugin](https://github.com/trustin/os-maven-plugin) and [osdetector-gradle-plugin](https://github.com/google/osdetector-gradle-plugin), I think it makes more sense to use `linux-x86_64-musl`. 

In the section titled "Customized deployments for specific releases of Linux" in their documentation, different variants of Linux are classified by appending identifiers to the end, like `linux-<arch>-debian`. For Alpine Linux, the value of `${os.detected.release.like}` is `alpine`. So the users of the `osdetector-gradle-plugin` can do something like the following to get auto-detect to work for alpine.
```
def getLibCVariant() {
    if (osdetector.release != null && osdetector.release.isLike('alpine')) {
        return '-musl'
    } else {
        return ''
    }
}

dependencies {
    implementation("software.amazon.awssdk.crt:aws-crt:0.22.5:${osdetector.classifier}${getLibCVariant()}")
}
```
I am not sure if it is this simple for os-maven-plugin but users can still achieve something similar by using profiles and taking a dependency on the following if `os.detected.release.like` is `alpine`
```
 <dependencies>
	<dependency>
	    <groupId>com.example</groupId>
	    <artifactId>example-native-lib</artifactId>
	    <version>${version}</version>
	    <classifier>${os.detected.classifier}-musl</classifier>
	</dependency>
</dependencies>
```